### DISCUSSION TITLE
Lab4 UserDMA問題

### DISCUSSION THREAD

**Discussion by tofu31** [bol-edu/HLS-SOC-Discussions#212] [Q&A]
請問Lab4需要修改DMA的部分嗎?
因為改這部分要重新跑HLS，可能有點麻煩

**Comment by michael61112** [comment]
目前的DMA裡面有針對EdgeDetect IP新加一些config，用不到可以不要去設定，用原本userDMA功能config就好，若你的FIR有需要一些新的config，可以去幫userDMA新加需要的config與功能
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/5982173a-49b8-41ab-a71c-8bf9900eb344)
